src/hooks/use-site-details.tsx

```diff
@@ -133,14 +133,24 @@ function useDeleteSite() {
 		async (
 			siteId: string,
 			removeLocal: boolean,
-			snapshots: Snapshot[]
+			snapshots: Snapshot[],
+			deleteSnapshots?: boolean
```

Comment:
[Question]: This is not used in the code above. Is it really necessary?
I mean, if the user is offline, and has more than one snapshot, this function will never be called.
Given that, it doesn't matter if the map below is run or not, as the array will be empty ( and will be resolved to an empty array ).

Reply:
Hey hey thanks for the feedback! Since we discussed this on Slack already, I simplified the logic. Let me know what you think!